gamegenerator.py

```diff
@@ -135,8 +137,8 @@ def __postConversionForEmuelec__(self):
         shutil.move(os.path.join(self.getLocalGameOutputDir()), emuElecDataDir)
         os.rename(os.path.join(emuElecDataDir, self.game + '.pc'), os.path.join(emuElecDataDir, self.game))
         # move *.bat *.map and *.cfg to pc/*.pc folder and rename *.cfg to dosbox-SDL2.conf
-        emuelecConfOutputDir = os.path.join(self.outputDir, 'pc', self.genre, self.game + ".pc") \
-            if self.useGenreSubFolders else os.path.join(self.outputDir, 'pc', self.game + ".pc")
+        emuelecConfOutputDir = os.path.join(self.outputDir, 'pc', self.genre, self.gameDir) \
```

Comment:
this should never be used for emuelec so this is not needed

Reply:
> this should never be used for emuelec so this is not needed

only for consistency